lib/engine/round/g_18_al/operating.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require_relative '../operating'
+
+module Engine
+  module Round
+    module G18AL
+      class Operating < Operating
+        def place_token(action)
+          super
+
+          ability = action.entity.all_abilities.find do |a|
```

Comment:
is cleaner

action.entity.abilities(:destination) do |ability|
 ...
end

Reply:
also make sure to call ability.remove so they can't use it twice

Reply:
Moved to g_18_al/token.rb. Modified to use :destination, and then remove_ability.